docs/README.md

```diff
@@ -144,7 +144,16 @@ An Operator is a user who helps run the software built on top of EigenLayer (AVS
 
 ##### Depositing Into EigenLayer
 
-Depositing into EigenLayer varies depending on whether the Staker is depositing Native ETH or LSTs:
+Depositing into EigenLayer varies depending on whether the Staker is depositing Natively Restaked beacon chain ETH or tokens.
+
+Depositing tokens includes the following high level steps:
+- StrategyManager.depositIntoStrategy().
+
+Depositing Natively Restaked beacon chain ETH includes the following high level steps:
+- EigenPodManager.createPod().
+- User modifies their validator withdrawal credentials to point to the EigenPod address.
+- Generate Proof via (eigenpod-proofs-generation)[https://github.com/Layr-Labs/eigenpod-proofs-generation] library.
+- EigenPod.verifyWithdrawalCredentials().
```

Comment:
See message in slack - would prefer flow-related changes to be PRed against the `feat/partial-withdrawal-batching` branch, since that's more up to date with the EigenPod redesign and will be merged into `dev` soon.